code/game/objects/items/crayons.dm

```diff
@@ -108,20 +108,19 @@
 		var/amount = weight * units_per_weight
 		reagents.add_reagent(reagent, amount)
 
-/obj/item/toy/crayon/proc/use_charges(amount)
+/obj/item/toy/crayon/proc/use_charges(mob/user, amount = 1, requires_full = TRUE)
 	// Returns number of charges actually used
-	switch(paint_mode)
-		if(PAINT_LARGE_HORIZONTAL)
-			amount *= 3
-
 	if(charges == -1)
 		. = amount
 		refill()
 	else
-		. = min(charges_left, amount)
-		charges_left -= .
+		if(check_empty(user, amount, requires_full))
+			return 0
```

Comment:
^

Reply:
But you never check that in the Repaint proc and color the suits anyway

Reply:
yeah, i'm just an idiot as always